roles/etcd/templates/etcd.j2

```diff
@@ -64,21 +72,9 @@ start)
     ;;
 
 stop)
-    {% if run_as == "worker" -%}
-    echo "==> no 'stop' action for proxy"
-    {% else -%}
```

Comment:
I think we need to keep this logic for the scenarios where we incrementally add members to the cluster and for `worker` nodes. May be we should add a `{% elif etcd_init_cluster -%}` block like for the `service start` case above and take no action in that block?